### DISCUSSION TITLE
oh no laragon detected exceptions! invalid class

### DISCUSSION THREAD

**Discussion by kilwayt** [leokhoa/laragon#690] [Q&A]
I HAVE PROBLEM EVERYTIME  WHEN I CLICK ON START ALL  BUTTON THEY SHOW ME 
oh no laragon detected exceptions! invalid class
THEN THEY CLOSS APP THE WINDOWS

**Comment by tomassue** [comment]
![image](https://github.com/leokhoa/laragon/assets/93255555/37a198ae-55b2-4701-9ef3-4b5fd6d57c97)

Once I click start all, this error appears. I'm using windows 10.

**Reply by Pen-y-Fan** [reply]
See #590 and #505 hopefully they will help.

**Comment by Boniich** [comment]
Hello! i had the same error, in my case, it was a port problem, something else is using the port 3306 of MySQL so Laragon and WAMP failed to run. 

I fix this error by changing the port of MySQL to 3307.

I hope this help to someone with the same error. Bye
